app/Commands/AwsEcsConnect.php

```diff
@@ -0,0 +1,271 @@
+            new InputOption('task', null, InputOption::VALUE_OPTIONAL, 'The task ID'),
+            new InputOption('container', null, InputOption::VALUE_OPTIONAL, 'Container name, ie php'),
+            new InputOption('command', "/bin/bash", InputOption::VALUE_OPTIONAL, 'The command to execute'),
+        ], $this->helpers->aws()->commonConsoleOptions()));
+    }
+
+    /**
+     * Execute the console command.
+     *
+     * @return mixed
+     */
+    public function handle()
+    {
+        $requiredBinaries = ['aws'];
+
+        if ($this->helpers->checks()->checkAndReportMissingBinaries($this, $requiredBinaries)) {
+            return 1;
+        }
+
+        $cluster = $this->option('cluster') ?: $this->askForCluster();
+
+        if (!$cluster) {
+            $this->error('No cluster provided.');
+            return 1;
+        }
+
+        $service = $this->option('service') ?: $this->askForService($cluster);
+
+        if (!$service) {
+            $this->error('No service provided.');
+            return 1;
+        }
+
+        $task = $this->option('task') ?: $this->askForTask($cluster, $service);
+
+        if (!$task) {
+            $this->error('No task provided.');
+            return 1;
+        }
+
+        $container = $this->option('container') ?: $this->askForContainer($cluster, $service, $task);
+
+        if (!$container) {
+            $this->error('No container provided.');
+            return 1;
+        }
+
+        $shellCommand = $this->option('command') ?: $this->askForCommand($cluster, $service, $task, $container);
+
+        if (!$shellCommand) {
+            $this->error('No command provided.');
+            return 1;
+        }
+
+        try {
+            $command = $this->helpers->aws()->ecs()->startCommandExecution($this, $cluster, $task, $container, $shellCommand);
+
+            $command->withTimeout(null)
+                ->echoLineByLineOutput(false)
+                ->echoOnFailure(false)
+                ->withProcessModifications(function ($process) {
+                    $process->setTty(Process::isTtySupported());
+                    $process->setIdleTimeout(null);
+                })
+                ->run();
+        } catch (ProcessFailed $e) {
+            $this->error("The SSM Agent likely isn't setup correctly on the selected container.");
+        }
+    }
+
+    protected function sendReRunHelper($rebuildOptions): void
+    {
+        $this->info("You can run this command again without having to go through options using this:");
+        $this->info(' ');
+        $this->comment("netsells aws:ssm:connect " . implode(' ', $rebuildOptions));
+        $this->info(' ');
+    }
+
+    protected function appendResolvedArgument($array, $key, $localValue = null): array
+    {
+        if ($this->option($key) || $localValue) {
+            $array[] = "--{$key}";
+            $array[] = $this->option($key) ?: $localValue;
+        }
+
+        return $array;
+    }
+
```

Comment:
Right now this is unused, so we could either delete it or add some code to show that help text.

```suggestion
```